packages/suite-data/files/translations/cs.json

```diff
@@ -1303,7 +1319,7 @@
   "TR_NETWORK_LITECOIN": "Litecoin",
   "TR_NETWORK_NAMECOIN": "Namecoin",
   "TR_NETWORK_NEM": "NEM",
-  "TR_NETWORK_OP": "Optimism",
+  "TR_NETWORK_OP": "Optimismus",
```

Comment:
:)

Reply:
I've fixed it in Crowdin already. Since Optimism will be behind Debug in 24.11 I will leave it there